src/SnD.Sdk/Kubernetes/KubernetesApiExtensions.cs

```diff
@@ -503,6 +503,95 @@ public static V1Job WithBillingId(this V1Job job, string billingId)
             });
         }
 
+        /// <summary>
+        /// Merges two V1PodFailurePolicies objects.
+        /// </summary>
+        /// <param name="ruleA"> A V1PodFailurePolicyRule object.</param>
+        /// <param name="ruleB"> A different V1PodFailurePolicyRule object.</param>
+        /// <returns> A V1PodFailurePolicyRule object.</returns>
+        public static V1PodFailurePolicyRule MergePodFailurePolicyRules(
+            V1PodFailurePolicyRule ruleA,
+            V1PodFailurePolicyRule ruleB)
+        {
+            if (ruleA.Action != ruleB.Action)
+            {
+                throw new InvalidOperationException("Cannot merge rules with different actions");
+            }
+
+            return new V1PodFailurePolicyRule(action: ruleA.Action,
+                onExitCodes: new V1PodFailurePolicyOnExitCodesRequirement(operatorProperty: "In",
+                    values: ruleA.OnExitCodes.Values.Concat(ruleB.OnExitCodes.Values).Distinct().ToList()),
+                onPodConditions: new List<V1PodFailurePolicyOnPodConditionsPattern>(
```

Comment:
Remove this...

Reply:
Just removed it in [b40566b](https://github.com/SneaksAndData/esd-services-sdk/pull/75/commits/b40566b9870ecd090376c23bb0876d75ad30da00) 👍